lib/src/blaze/fetch_full_transaction.rs

```diff
@@ -577,62 +464,163 @@ impl FetchFullTxns {
             wallet_transactions,
             is_outgoing_transaction,
             outgoing_metadatas,
-            Keys::okeys,
-            |t: &Transaction| t.orchard_bundle(),
-            orchard::Bundle::actions,
-            |key| OrchardIvk::try_from(&key.key),
-            OrchardDomain::for_action,
-            <[u8; 512]>::as_slice,
         )
         .await;
     }
 }
 
-async fn scan_bundle<K, B, L, O, D, E>(
-    _config: &ZingoConfig,
+async fn scan_bundle<D>(
+    config: &ZingoConfig,
     transaction: &Transaction,
-    _height: BlockHeight,
-    _unconfirmed: bool,
-    _block_time: u32,
+    height: BlockHeight,
+    unconfirmed: bool,
```

Comment:
We don't want this `unconfirmed` to be decouplable from this transaction.  Maybe we need a:
`TransactionAndMetadata` type-or-trait?

Reply:
Issued.